src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -492,8 +490,7 @@ private void transactionRoutes(){
                     exchange.getIn().setBody(response.toString());
                 });
 
-        from("rest:POST:/channel/transaction/{" + TRANSACTION_ID + "}/resolve")
-                .id("transaction-resolve")
+        from("rest:POST:/channel/transaction/{" + TRANSACTION_ID + "}/resolve")                .id("transaction-resolve")
```

Comment:
recheck if you forget to migrate this API.

Reply:
Sorry I forgot but now done!